src/DB_Store.php

```diff
@@ -45,6 +78,42 @@ public function save_action( ActionScheduler_Action $action, \DateTime $date = n
 		}
 	}
 
+	/**
+	 * Update an existing action by ID.
+	 *
+	 * @author Jeremy Pry
+	 *
+	 * @param int   $action_id The action ID to update.
+	 * @param array $fields    The array of field data to update.
+	 *
+	 * @return false|int
+	 */
+	public function update_action( $action_id, array $fields ) {
```

Comment:
A generic update method like this makes to add.

What do you think of implementing it on `ActionScheduler_Store` as well so it can be relied on by all data stores?

It will add a little extra work, as it will need to be implemented in `ActionScheduler_wpPostStore` also, but we're already breaking backward compatibility in the next version of AS as `ActionScheduler_Store` will require child classes to implement other new methods, like `action_counts()`, `get_claim_id()` and `get_status()`. So it seems like a good time to add it if we think it will be required in future.